src/main/java/org/folio/dew/service/JobCommandsReceiverService.java

```diff
@@ -188,7 +188,8 @@ private boolean deleteOldFiles(JobCommand jobCommand, Acknowledgment acknowledgm
   }
 
   public void addBulkEditJobCommand(JobCommand jobCommand) {
-    bulkEditJobCommands.put(jobCommand.getId().toString(), jobCommand);
+    if (!bulkEditJobCommands.containsKey(jobCommand.getId().toString()))
+      bulkEditJobCommands.put(jobCommand.getId().toString(), jobCommand);
```

Comment:
I think it's better to use `putIfAbsent(k, v)` method here

Reply:
Fixed, thanks.